Changelog

```diff
@@ -1,3 +1,45 @@
+Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.0.0
+    - FEATURE: Drop support for Qt5, qmake, autotools, Windows < 10, Windows 32-bit
```

Comment:
It looks strange to me that such changes are mentioned as a FEATURE.

Reply:
Will be downgraded to `OTHER`